src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php

```diff
@@ -86,6 +88,19 @@ protected static function bootKernel(array $options = [])
         return static::$kernel;
     }
 
+    /**
+     * This is the best way to get a dependency injection container. It has access
+     * to public and private services.
+     */
+    protected static function getContainer(): ContainerInterface
```

Comment:
maybe `bool $reboot = false`. but time will tell :)

Reply:
Rebooting is rare, I dont think that would be useful. 

But we will see, it can be added later as you suggest.